emcc.py

```diff
@@ -1094,6 +1094,12 @@ def get_last_setting_change(setting):
     # Apply -s settings in newargs here (after optimization levels, so they can override them)
     apply_settings(settings_changes)
 
+    if shared.Settings.MINIMAL_RUNTIME or 'MINIMAL_RUNTIME=1' in settings_changes:
```

Comment:
why check both conditions here? I think just the first should be enough.

Reply:
The thinking here is that removal of these default functions this needs to be applied before user passes -s DEFAULT_LIBRARY_FUNCS_TO_INCLUDE=[] params from command line, hence reading `'MINIMAL_RUNTIME=1' in settings_changes` directly. (now I realize it also needs to look for `MINIMAL_RUNTIME=2` from cmdline) Because if user wants to force-export e.g. malloc and they specify `-s DEFAULT_LIBRARY_FUNCS_TO_INCLUDE=[malloc]` on command line, this block would override user's setting and still remove it.

The explicit check for `shared.Settings.MINIMAL_RUNTIME` is because to test MINIMAL_RUNTIME against the test suite, I go and flip the default value in src/settings.js to get all code building with that setting enabled by default.